PolyEngine/Engine/Src/Resources/MeshResource.cpp

```diff
@@ -150,9 +150,10 @@ void MeshResource::SubMesh::LoadBones(aiMesh* mesh)
 	if (mesh->HasBones())
 	{
 		ASSERTE((i8)mesh->mNumBones <= std::numeric_limits<typename decltype(MeshData.BoneIds)::value_type::ValueType>::max(), "Model has too many bones!");
-
-		std::vector<::pe::core::storage::PriorityQueue<std::pair<u8, float>, std::function<bool(const std::pair<u8, float>&, const std::pair<u8, float>&)>>> tmpBonesList;
-		tmpBonesList.resize(mesh->mNumVertices, { [](const std::pair<u8, float>& v1, const std::pair<u8, float>& v2) { return v1.second > v2.second; } });
+	
+		std::vector<std::priority_queue<std::pair<u8, float>,std::vector<std::pair<u8, float>>, std::function<bool(const std::pair<u8, float>&, const std::pair<u8, float>&)>>> tmpBonesList;
```

Comment:
Wow, that's a mouthful. Nice job though.

Reply:
how about usings for this one?

Reply:
That's actually a good idea. Let's do it.